src/LinAlg/hiopMatrixRajaSparseTriplet.cpp

```diff
@@ -1088,8 +1122,9 @@ void hiopMatrixRajaSparseTriplet::copyRowsBlockFrom(const hiopMatrix& src_gen,
     src.row_starts_ = src.allocAndBuildRowStarts();
   }
   assert(src.row_starts_);
-  index_type* src_row_st_host = src.row_starts_->idx_start_host_;
 
+#if 0 // cpu code due to the RAJA::scan bug
```

Comment:
We can always look in the git history to find the cpu implementation if we need it, I recommend we remove this ifdef-ed block.